inc/functions/cloudflare.php

```diff
@@ -737,6 +737,17 @@ function rocket_auto_purge_cloudflare_by_url( $post, $purge_urls, $lang ) {
 		return;
 	}
 
+	// Add home URL and feeds URLs to Cloudflare clean cache URLs list.
+	$home_url     = get_rocket_i18n_home_url( $lang );
+	$purge_urls[] = $home_url;
```

Comment:
As `$home_url` is not used elsewhere in this function, we can combine these 2 lines of code:

```php
$purge_urls[] = get_rocket_i18n_home_url( $lang );
```